locale/en_US/editor.po

```diff
@@ -272,6 +272,9 @@ msgstr "Unconsider this Review"
 msgid "editor.review.readConfirmation"
 msgstr "Once this review has been read, press \"Confirm\" to indicate that the review process may proceed. If the reviewer has submitted their review elsewhere, you may upload the file below and then press \"Confirm\" to proceed."
 
+msgid "editor.review.discussion.errorAnonymousParticipants"
+msgstr "You can not create a discussion that includes several anonymous reviewers, or the author and an anonymous reviewer, because the person's identity will be revealed."
```

Comment:
Good! Tiny language adjustment:

> A discussion can not be created with the selected participants because it would impact the anonymity of the review process.